assets/sample-orchestration-request.json

```diff
@@ -0,0 +1,24 @@
+{
+  "xml_message": "",
+  "xml_message_type": "ecr",
```

Comment:
I'm curious about this one - what are you envisioning `xml_message`/`xml_message_type` will be used for?

Reply:
I'm guessing `xml_message` is the field for the raw message to be sent into the FHIR converter? Though we also support HL7 messages which are pipe delimited files, so maybe it should just be `message` and `message_type`?

Reply:
Ahhhh gotcha, that makes sense! I think in that case my vote would be to make those part of the request params for our `parse` endpoint, vs part of the configuration. Thanks for explaining!